command.py

```diff
@@ -0,0 +1,24 @@
+#This file in its current state only exists for reference
+class Command:
```

Comment:
Delete out ALL commands you are not currently using. The point of developing a GUI in the `MVP` way is to allow it to grow organically. We don't project future requirements, and especially not bits of code until we actually need them.